### ISSUE TITLE
Input for the -r argument 

### ISSUE THREAD

**Issue by tyler-hansen** [ReddyLab/CRADLE#26]
Hi Reddy Lab, 

This is a cool tool, so thank you! I didn't find a bug, but I had a question about the -r option. 

I'm a little confused about what file I am supposed to supply for -r because it seems like these are the regions you refine the analysis to? In your paper, which was great, you say you binned the genome into windows using (1.5*L) as the bin size. Would the resulting regions serve as the -r argument or does that binning step happen within the tool?  

In other words, what did you use as the -r option in the paper? Is it as simple as supplying something like a modified chrom sizes file?

I could be totally off here too, so sorry and let me know. 

Thank you!

Best, Tyler

**Comment by Young-Sook** [comment]
Hi Tyler,

-r parameter is to define target regions. In our manuscript, both of the data we used are whole genome STARR-seq, so we used whole genome (except sex chromosomes) as target regions. Below is the file we used for -r. 
```
chr1	1	248956422
chr2	1	242193529
chr3	1	198295559
chr4	1	190214555
chr5	1	181538259
chr6	1	170805979
chr7	1	159345973
chr8	1	145138636
chr9	1	138394717
chr11	1	135086622
chr10	1	133797422
chr12	1	133275309
chr13	1	114364328
chr14	1	107043718
chr15	1	101991189
chr16	1	90338345
chr17	1	83257441
chr18	1	80373285
chr20	1	64444167
chr19	1	58617616
chr22	1	50818468
chr21	1	46709983
```

We wanted to avoid calling peaks in the black list regions so we used -bl parameter as well. Here is the first few lines of the file we used for -bl. 
```
chr1 0	10000
chr1	207666	257666
chr1	297968	347968
chr1	535988	585988
```

CRADLE callPeak subcommand bins the regions provided in -r parameter, using a window of which length is defined by -rbin.  We originally used 1.5*L as the default value for -rbin, but we recently decided to use fixed bin size (300bp; you can tune this parameter in -rbin) as default to avoid confusion. 

Let me know if you have any further question.

Thanks,
Young-Sook

**Comment by tyler-hansen** [comment]
Thank you! I'll give this a shot. I'm really looking forward to it.
